lib/irb/context.rb

```diff
@@ -154,13 +151,6 @@ def initialize(irb, workspace = nil, input_method = nil)
 
     def save_history=(val)
```

Comment:
I don't think we need to have this `save_history` accessors on individual context objects, but it's better to keep them for compatibility.